packages/terra-dialog-modal/src/DialogModal.scss

```diff
@@ -0,0 +1,91 @@
+:local {
+  .dialog-modal-wrapper {
+    background-color: var(--terra-dialog-modal-color, #fff);
+    border-radius: var(--terra-dialog-modal-border-radius, 0.3125rem);
+
+
+    &.width-320 {
+      width: 320px;
+    }
+
+    &.width-480 {
+      width: 480px;
+    }
+
+    &.width-640 {
+      width: 640px;
+    }
+
+    &.width-800 {
+      width: 800px;
+    }
+
+    &.width-960 {
+      width: 960px;
+    }
+
+    &.width-1120 {
+      width: 1120px;
+    }
+
+    &.width-1280 {
+      width: 1280px;
+    }
+
+    &.width-1440 {
+      width: 1440px;
+    }
+
+    &.width-1600 {
+      width: 1600px;
+    }
+
+    &.width-1760 {
+      width: 1760px;
+    }
+
+    &.width-1920 {
+      width: 1920px;
+    }
+
+    @media screen and (max-width: 768px) {
```

Comment:
The viewport width `768px` is not a Terra-defined viewport. I'm curious, what determined the max-width of `max-width: 768px`?

Reply:
Just referred it here. I can change it to something else. 
https://github.com/cerner/terra-framework/blob/408cb9c6ab9c3aa7887afc8938e777b1f1fc4049/packages/terra-slide-panel/src/SlidePanel.scss#L92